scripts/build.ts

```diff
@@ -1,6 +1,11 @@
-import { execSync as Child } from 'child_process';
+import { execSync } from 'child_process';
 import FS from 'fs';
 
+const Child = (command: string) => {
```

Comment:
All the script files really ought to be translated into a scripting language, or at least downgraded to regular JavaScript. I hate that JS in this case is _the_ portable solution, but it doesn't matter much because it is a thinly veiled wrapper around `gnutils`. It's only used for native JSON parsing to generate the `dist/npm` package config